src/views/Home.vue

```diff
@@ -1,8 +1,17 @@
 <template>
   <div id="home">
     <hero></hero>
-
+    <h2>
+      Listen to the assholes ramble
+    </h2>
     <sound-cloud></sound-cloud>
+  <h2>
+    You sing and you sing good!
+  </h2>
+    <h3>
+      (on video)
```

Comment:
This is just plain text and it's not an actual YouTube video component are you missing commits? That you haven't pushed up yet?